Modules/posixmodule.c

```diff
@@ -8263,42 +8241,25 @@ os_setpgrp_impl(PyObject *module)
 #ifdef HAVE_GETPPID
 
 #ifdef MS_WINDOWS
-#include <tlhelp32.h>
+#include <processsnapshot.h>
 
 static PyObject*
 win32_getppid()
 {
-    HANDLE snapshot;
     PyObject* result = NULL;
-    BOOL have_record;
-    PROCESSENTRY32 pe;
-
-    DWORD mypid = GetCurrentProcessId(); /* This function never fails */
+    HANDLE myhandle = GetCurrentProcess();
```

Comment:
I'd prefer `hprocess` or `process` instead of the generic name `myhandle` (to what?).